server/conf/messages.ko

```diff
@@ -34,6 +34,12 @@ validation.isRequired=이 질문은 필수입니다.
 # Validation error that is shown when the user input couldn't be converted for any reason.
 validation.invalidInput=유효한 내용을 입력하세요.
 
+# Error message announced to screen reader when there are errors on the current page.
+validation.errorAnnouncementSr=양식에 오류가 있습니다. Please fix before continuing.
```

Comment:
NB: Did you mean to have the "Please fix..." English part here?

Reply:
Nope, good catch!